chemprop/cli/predict.py

```diff
@@ -6,14 +6,11 @@
 
 from lightning import pytorch as pl
 import torch
+from lightning.pytorch.loggers import TensorBoardLogger
```

Comment:
Without specifying the logger, the tensorboard log files defaulted to saving in `lightning_logs`. This change is similar to in `train.py`.

Reply:
Interestingly `pl.Trainer(logger=None)` defaults to using the tensorboard logger and produces files like `events.out.tfevents.1705698880.login-2` in `lightning_logs/`. I added it in explicitly so that I could change where these logs get put. 

But the real solution is to use `pl.Trainer(logger=False)` to not log. I'll implement that.